zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -19,14 +19,7 @@
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.net.UnknownHostException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Queue;
-import java.util.Set;
+import java.util.*;
```

Comment:
Thanks @reminia. 
Like @felixcheung mentioned, we're following [style](http://zeppelin.apache.org/contribution/contributions.html#code-convention) which discourages [whilecard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

(style guide is not easy to find. i think we need to automate style check using stylecheck plugin)

 @reminia could you try resolve issue without using `*`?
Once it's updated, i'll merge this to branch-0.7 immediately as a hotfix.